chat/fe/src/views/RoomView.ts

```diff
@@ -15,54 +23,62 @@ type RoomViewProps = {
 };
 
 export function RoomView(props: RoomViewProps) {
+  let messages: MessageType[] = [];
+  let userRoomConfig: {
+    lastReadMessageId: string;
+  } = null;
+
   const roomView = Div();
 
   setStyle(roomView, {
-    display: "flex",
-    flexDirection: "column",
-    height: "calc(100vh - 50px)",
-    flexGrow: "1",
+    display: 'flex',
+    flexDirection: 'column',
+    height: 'calc(100vh - 50px)',
+    flexGrow: '1',
   });
 
   const room = getRoom(props.roomId);
 
   onRoomMessage(props.roomId, (message) => {
+    if (isLastMessageToday(messages[1], message)) {
```

Comment:
This seems weird to be using messages[1].  I would think this should be messages[messages.length - 1].  I'm going to merge, since I think the rest looks good and I assume this only causes issues on new message.  If it's incorrect we can fix separately.